clients/smtpNotifier.go

```diff
@@ -0,0 +1,52 @@
+package clients
+
+import (
+	"log"
+	"net/smtp"
+)
+
+type (
+	// SmtpNotifier
+	SmtpNotifier struct {
+		Config *SmtpNotifierConfig
+	}
+
+	// SmtpNotifierConfig contains the static configuration for the smtp service
+	// Credentials come from the environment and are not passed in via configuration variables.
+	SmtpNotifierConfig struct {
+		From     string `json:"fromAddress"`
+		Server   string `json:"serverAdress"`
+		User     string `json:"user"`
+		Password string `json:"password"`
+	}
+)
+
+//NewSmtpNotifier creates a new SMTP notifier (using standard smtp to send emails)
+func NewSmtpNotifier(cfg *SmtpNotifierConfig) (*SmtpNotifier, error) {
+	return &SmtpNotifier{
+		Config: cfg,
+	}, nil
+}
+
+// Send a message to a list of recipients with a given subject
+func (c *SmtpNotifier) Send(to []string, subject string, msg string) (int, string) {
+	// Set up authentication information.
+	var auth smtp.Auth
+	// If not user is provided, then do not try to authenticate to the server (for dev only)
+	if c.Config.User != "" {
+		auth = smtp.PlainAuth("", c.Config.User, c.Config.Password, c.Config.Server)
```

Comment:
We pass the same parameter c.Config.Server to the PlainAuth (host parameter) and SendMail (addr parameter) functions. 
func PlainAuth(identity, username, password, host string)
func SendMail(addr string, a Auth, from string, to []string, msg []byte) error

But host in PlainAuth needs an host without a port and addr in SendMail needs a full address with port (mandatory). 

Either split the c.Config.Server with ":" to retrieve only the host for PlainAuth , or pass the host and port as two config parameters.